CONTRIBUTING.md

```diff
@@ -2,6 +2,36 @@
 
 If you're reading this, you're awesome! Thank you for helping us make this project great and being a part of the Material-UI community. Here are a few guidelines that will help you along the way.
 
+## Git workflow
+
+1. Fork the Material-UI repository on GitHub
```

Comment:
I think we wanted a link to a basic github-fork-clone-pr workflow?

https://github.com/mui-org/material-ui/pull/17562/files#diff-6a3371457528722a734f3c51d9238c13R13

Reply:
https://github.com/mui-org/material-ui/pull/17597#issuecomment-536091610 Check why not. I checked that document in detail, and I think that that document can only confuse when you have all the needed explanations in the bullets already.